add_duo_schema_bindings.py

```diff
@@ -0,0 +1,44 @@
+import asyncio
+from synapseclient.models import Project, File
+import synapseclient
+import json
+import os
+
+import yaml
+
+
+def main():
+    """Invoke adding DUO schema bindings"""
+    syn = synapseclient.Synapse()
+    # syn.setEndpoints(**synapseclient.client.STAGING_ENDPOINTS)
+    syn.login()
+    with open("config.yml", "r") as f:
+        config = yaml.safe_load(f)
+
```

Comment:
Add another script to select the DUO codes, and to auto generate the json schema template